### PR TITLE
Story

### PR DESCRIPTION
**Pull request by Mexaby**

### CODE REVIEW

**Review by Mexaby — COMMENTED** [review]
Seems alright

**Comment by Mexaby** [comment]
Can't approve my own request ¯\_(ツ)_/¯

File: C#Basics/AnimalRescue/VetAssistant.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AnimalRescue
+{
+    internal class VetAssistant
+    {
```
Comment (ramonaZmole) [inline C#Basics/AnimalRescue/VetAssistant.cs:10]: a simpler way to setup object properties is setting the properties like this 
image

Having a constructor with all the properties as a parameter is a bad practice.

When you initialize the object you set the properties like this 
image

Please adjust everywhere where you have constructors

Comment (Mexaby) [reply C#Basics/AnimalRescue/VetAssistant.cs:10]: Will do.

**Comment by clauu-mara** [comment]
I will leave here a comment related to the OOP assignment.
- you don't need to create another private int hunger into the Cat class if it is already existing in base class 
image
- I didn't find the AnimalFood/Vet/ RehabilitationActivity/ DogFood classes here in git in the c#Basics folder - I don't know why
- the attributes that are general (for all the animals should be added to the base Class - Animal)
- you should use setters/getters - it is a good practice than using the constructor
- you didn't use overloading

**Comment by Mexaby** [comment]
Won't do, way past its time.

**Pull request closed without merging**